.github/workflows/debezium-workflow.yml

```diff
@@ -813,6 +820,7 @@ jobs:
           -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
           -Dmaven.wagon.http.pool=false
           -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
+          -Dfail.flaky.tests=false
```

Comment:
@Naros Is this correct here?

Also I am afraid we have know workflows that use `fail.flaky.tests` control flag - like Db2 and maybe others. We need to align on the same behaviour.